guides/testing-hypothesis.rst

```diff
@@ -70,7 +70,7 @@ The following will give you a working virtualenv for running tests in:
   pip install -e .
 
   # Test specific dependencies.
-  pip install pytest-xdist flaky mock pexpect
+  pip install -r requirements/test.in
```

Comment:
This file doesn't seem to have the same contents as the previous command.

Reply:
Also, should it be the pinned version? .txt instead of .in?

Reply:
That's the idea - the previous command was badly out of date.  We don't use `flaky` at all anymore, and `mock` is only required on Python 2!